api-controllers/ServiceFabrikAdminController.js

```diff
@@ -3,20 +3,20 @@
 const _ = require('lodash');
 const Promise = require('bluebird');
 var moment = require('moment-timezone');
-const catalog = require('../models/catalog');
-const errors = require('../errors');
-const logger = require('../logger');
-const config = require('../config');
-const NetworkSegmentIndex = require('../bosh/NetworkSegmentIndex');
-const backupStore = require('../iaas').backupStore;
+const catalog = require('../broker/lib/models/catalog');
+const errors = require('../common/errors');
+const logger = require('../common/logger');
+const config = require('../common/config');
+const NetworkSegmentIndex = require('../data-access-layer/bosh/NetworkSegmentIndex');
+const backupStore = require('../data-access-layer/iaas').backupStore;
 const FabrikBaseController = require('./FabrikBaseController');
-const utils = require('../utils');
-const fabrik = require('../fabrik');
-const bosh = require('../bosh');
-const ScheduleManager = require('../jobs');
-const BackupReportManager = require('../reports');
-const CONST = require('../constants');
-const maintenanceManager = require('../maintenance').maintenanceManager;
+const utils = require('../broker/lib/utils');
+const fabrik = require('../broker/lib/fabrik');
```

Comment:
Some of the parts like **broker/lib/fabrik** can be renamed to **service/fabrik**. Or some other sub directories also. 
So, we can look package structure as **controllers, service, dal**.

Reply:
This is only first part of the restructuring. Avoided only controllers as it also clashes with Kubernetes controllers. api-controllers is more explicit I guess.

Reply:
The Fabrik folder would slowly go away as all the codes would move to managers.

We would remove the broker folder and have everything either directly under root or under different folders. I’ll give a final folder structure snippet that we agreed on in the last meeting in the description.